uisdk/src/main/res/layout/uisdk_view_loyalty_view.xml

```diff
@@ -3,63 +3,89 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:animateLayoutChanges="true"
-    android:orientation="vertical">
+    android:orientation="vertical"
+    android:background="@color/kh_uisdk_background_primary"
+    >
 
     <LinearLayout
-        android:id="@+id/loyaltyViewLayout"
-        style="@style/KhLoyaltyLayoutView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:paddingStart="@dimen/kh_uisdk_spacing_small_intermediate"
-        android:paddingTop="@dimen/kh_uisdk_padding_default"
-        android:paddingEnd="@dimen/kh_uisdk_spacing_small_intermediate"
-        android:paddingBottom="@dimen/kh_uisdk_padding_default">
+        android:orientation="vertical"
+        android:layout_marginTop="@dimen/kh_uisdk_loyaltyViewBalance_marginTop"
+        >
 
+        <TextView
+            android:id="@+id/loyaltyViewBalance"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            style="@style/KhLoyaltyBalanceView"
+            android:layout_gravity="end"
+            android:paddingTop="@dimen/kh_uisdk_loyaltyViewBalance_paddingTop"
```

Comment:
These attributes can also be moved into `KhLoyaltyBalanceView`, any reason not to do it?